guides/common/modules/proc_restricting-custom-repositories-to-a-specific-os-version-or-architecture.adoc

```diff
@@ -0,0 +1,20 @@
+[id="Restricting_Custom_Repositories_to_a_Specific_OS_Version_{context}"]
+= Restricting Custom Repositories to a Specific Operating System or Architecture in {Project}
+
+You can configure {Project} to make custom repositories available only on hosts with a specific operating system version or architecture.
+For example, you can restrict custom repositories only to {RHEL}{nbsp}9 hosts.
+
+ifdef::satellite[]
+[NOTE]
+Only restrict architecture and operating system version for custom products.
+{Project} applies these restrictions automatically for Red{nbsp}Hat repositories.
+endif::[]
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Products*.
+. Click the product that contains the repository sets you want to restrict.
+. In the *Repositories* tab, click the repository you want to restrict.
+. Navigate to *Publishing Settings* > *Restrict to OS version* to restrict the operating system version.
```

Comment:
```suggestion
. In the *Restrict to OS version* list, select the operating system version you want to restrict content to.
```

Unsure about "you want to ...".

my main point: Please don't use "navigate to *A* > *B*" again; we only use this for the navigation.

Reply:
Ok, thanks! I would like to keep the "Publishing Settings" section of the page, I believe it would help users find the required configuration options faster.

What would you think about this:

```
. In the *Publishing Settings* section, set the following options:
+
* Set *Restrict to OS version* to restrict the operating system version.
+
* Set *Restrict to architecture* to restrict the architecture.

```
I'm aware that this still uses "+\n", but this time, it's only a list of options, not separate actions like before.

Reply:
We don't generally mention subheadings AFAIK. I've liked it as was before; but I am OK with this change. Other thoughts?

Reply:
I wanted to include the "Publishing Settings" subheading because it was a useful 'landmark' in the long list of options on that web UI page as I was clicking through the procedure myself. But that can be very subjective.

I'll keep this thread open for a while and if nobody else chimes in, I'll keep it as it is.

Reply:
No further comments, so I'm resolving this.